src/olympia/addons/models.py

```diff
@@ -1545,6 +1545,16 @@ def type_can_set_compatibility(cls, addon_type):
     def can_set_compatibility(self):
         return self.type_can_set_compatibility(self.type)
 
+    @property
+    def can_be_compatible_with_all_fenix_versions(self):
+        """Whether or not the addon is allowed to be compatible with all Fenix
+        versions (i.e. it's a recommended/line extension for Android)."""
+        return (
+            self.promoted
+            and self.promoted.group.can_be_compatible_with_all_fenix_versions
```

Comment:
you could do the following to save a line but I'm not sure it's worth it.
```suggestion
            self.promoted_group.can_be_compatible_with_all_fenix_versions
```